spring-boot-starter-camunda-sdk/src/main/java/io/camunda/zeebe/spring/client/properties/ZeebeClientConfigurationProperties.java

```diff
@@ -805,10 +805,10 @@ public String toString() {
           + clusterId
           + '\''
           + ", clientId='"
-          + clientId
+          + "***"
```

Comment:
👍 I like the pragmatic solution

❓ I understand we redact the `clientSecret` as it is a secret. But the `clientId` is not really secret information and may help troubleshoot configurations. Is there a specific reason we're also redacting the `clientId`?

Reply:
We are doing [that](https://github.com/camunda/zeebe/blob/0d2e6d277efb00b4f3ca932764f025470c578259/spring-boot-starter-camunda-sdk/src/main/java/io/camunda/zeebe/spring/client/properties/common/Client.java#L35-L36) for SM clusters, so I thought to have the same behaviour also for Saas

Reply:
Ah, I see. Thanks @nicpuppa 👍 I guess it doesn't hurt too much if we have clear config properties and documentation. Makes sense to do it consistently.